test/FunctionalTests/Balancer/DnsResolverTests.cs

```diff
@@ -0,0 +1,52 @@
+﻿#region Copyright notice and license
+
+// Copyright 2019 The gRPC Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#endregion
+
+#if NET5_0_OR_GREATER
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Grpc.AspNetCore.FunctionalTests;
+using Grpc.Net.Client.Balancer;
+using NUnit.Framework;
+
+namespace Grpc.AspNetCore.FunctionalTests.Balancer
+{
+    [TestFixture]
+    public class DnsResolverTests : FunctionalTestBase
+    {
+        [Test]
+        public async Task RefreshAsync_OneSubscriber_HasResult()
```

Comment:
only 1 test for DnsResolver doesn't seem sufficient? Especially since this is going to be the most commonly used resolver.

Reply:
Done